harness/determined/cli/config_policies.py

```diff
@@ -8,52 +8,86 @@
 
 def describe_config_policies(args: argparse.Namespace) -> None:
     sess = cli.setup_session(args)
+
     workload_type = "EXPERIMENT"
-    if args.workload_type.upper() == "NTSC":
+    if args.workload_type.upper() == "TASKS":
         workload_type = "NTSC"
 
-    wksp = api.workspace_by_name(sess, args.workspace)
-    resp = bindings.get_GetWorkspaceConfigPolicies(
-        sess, workloadType=workload_type, workspaceId=wksp.id
-    )
+    if args.workspace_name:
+        wksp = api.workspace_by_name(sess, args.workspace_name)
+        wksp_resp = bindings.get_GetWorkspaceConfigPolicies(
+            sess, workloadType=workload_type, workspaceId=wksp.id
+        )
+        if args.json:
```

Comment:
I'm looking for something like `cli.Arg("--json", action="store_true", help="print as JSON")`. Is this still required?

Reply:
Yea! We have that in the `describe` `CLI.Arg`!

Reply:
Ah thanks I missed it.

Reply:
np!